src/components/Calculator.js

```diff
@@ -51,51 +44,47 @@ class Calculator extends Component {
     }
 
     return result;
-  }
+  };
 
-  render() {
-    const { calculatorObj } = this.props;
-    return (
-      <div className="table">
-        <div className="row field">
-          <p id="field-text">
-            { this.displayResult() }
-          </p>
-        </div>
-        <div className="row">
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, 'AC'); }}>AC</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '+/-'); }}>+/-</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '%'); }}>%</div>
-          <div className="button orange" onClick={() => { this.calculate(calculatorObj, '÷'); }}>÷</div>
-        </div>
-        <div className="row">
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '7'); }}>7</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '8'); }}>8</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '9'); }}>9</div>
-          <div className="button orange" onClick={() => { this.calculate(calculatorObj, 'x'); }}>X</div>
-        </div>
-        <div className="row">
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '4'); }}>4</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '5'); }}>5</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '6'); }}>6</div>
-          <div className="button orange" onClick={() => { this.calculate(calculatorObj, '-'); }}>-</div>
-        </div>
-        <div className="row">
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '1'); }}>1</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '2'); }}>2</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '3'); }}>3</div>
-          <div className="button orange" onClick={() => { this.calculate(calculatorObj, '+'); }}>+</div>
-        </div>
-        <div className="row">
-          <div className="big-button gray" onClick={() => { this.calculate(calculatorObj, '0'); }}>0</div>
-          <div className="button gray" onClick={() => { this.calculate(calculatorObj, '.'); }}>.</div>
-          <div className="button orange" onClick={() => { this.calculate(calculatorObj, '='); }}>=</div>
-        </div>
+  return (
+    <div className="table">
+      <div className="row field">
+        <p id="field-text">
```

Comment:
- [ ] When I push any number the default zero disappears and doesn't appear in the numbers I pushed, please work on it. 🐶 
![image](https://user-images.githubusercontent.com/20563252/133688763-4beab8c9-7a4a-4a2d-8b1c-943a6ba02831.png)